dependent/chemmedchem.csl

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>ChemMedChem</title>
+    <id>http://www.zotero.org/styles/chemmedchem</id>
+    <link href="http://www.zotero.org/styles/chemmedchem" rel="self"/>
+    <link href="http://www.zotero.org/styles/angewandte-chemie" rel="independent-parent"/>
+    <link href="https://chemistry-europe.onlinelibrary.wiley.com/hub/journal/18607187/notice-to-authors" rel="documentation"/>
+    <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/manuscript-guidelines-1.0.pdf" rel="documentation"/>
```

Comment:
Is the springer documentation linked on purpose?

Reply:
Mit dem hin und her wegen der Validierung hab ich den dann am Schluss vergessen. Danke.